pages/contribute/join.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Join the Community Group
+---
+
+The best way to stay informed about the activities of ACT-Rules Community is to become a member.
+
+## Mailing list keeps you up to date
+
+By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if no feedback
```

Comment:
I'm not sure about the reference to Github here. As in, it might scare people away feeling they need to know what is Github and what is a pull request in order to join the CG.
```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes (new rules, correction to existing ones, …) that have been approved by at least 3 other members, and will be published once there is no unresolved feedback.
```

Maybe also add a rough estimate of the number of emails (2-3/week?) to avoid scaring people in fear of being "spammed" (the list here sounds like there is a lot happening on the ML, but it is not really the case).

Reply:
That's a good suggestion. Although it may still be worth mentioning that those announced changes may be viewed on the Github without needing an account (which I think it true).